js/rpg_core/Sprite.js

```diff
@@ -222,10 +227,16 @@ Sprite.prototype.setColorTone = function(tone) {
  * @private
  */
 Sprite.prototype._onBitmapLoad = function() {
-    if (this._frame.width === 0 && this._frame.height === 0) {
-        this._frame.width = this._bitmap.width;
-        this._frame.height = this._bitmap.height;
+    if(this._requestBitmap === this._bitmap){
```

Comment:
Of course it's async, so bitmap may be changed.
But... `this._requestBitmap` is changed at the same time. (69-70)
So, this condition-expression will always be `true`.